compiler_opt/rl/feature_ops.py

```diff
@@ -42,7 +42,8 @@ def build_quantile_map(quantile_file_dir: str):
 
 def discard_fn(obs: types.Float):
```

Comment:
@yundiqian note the change to this function that I just added - it is similar to the change to ConstantValueNetwork to avoid relying on static tensor shapes (obs.shape) and instead use the dynamic tensor shape. This has a slight functional change as the discarded tensor shapes now have a final dimension of size 1 instead of 0, but the input data is still zeroed out. The only observable difference is a change in the signature of the model after the preprocessing layers.